src/main/java/cloud/fogbow/ras/api/http/request/Compute.java

```diff
@@ -34,6 +34,21 @@
 
     private final Logger LOGGER = Logger.getLogger(Compute.class);
 
+    @RequestMapping(value = "/{computeId}/snapshot", method = RequestMethod.POST)
+    public ResponseEntity<Boolean> takeSnapshot(
+            @PathVariable String computeId,
+            @RequestBody String name,
+            @RequestHeader(required = false, value = CommonKeys.SYSTEM_USER_TOKEN_HEADER_KEY) String systemUserToken) throws FogbowException{
+        try {
+            ApplicationFacade.getInstance().takeSnapshot(computeId, systemUserToken, ResourceType.COMPUTE, name);
```

Comment:
To keep consistent with the other calls this method signature should be:
takeSnapshot(computeId, name, systemUsertoken, ResourceType.COMPUTE);